docs/intro/contributing.txt

```diff
@@ -91,7 +91,7 @@ Installing Git
 ==============
 
 For this tutorial, you'll need Git installed to download the current
-development version of Django and to generate patch files for the changes you
+development version of Django and to generate a pull request for the changes you
```

Comment:
This looks great but it needs to be wrapped at 79cols. Could you please adjust this sentence with that in mind?